### ISSUE TITLE
Flaky failures in tests

### ISSUE THREAD

**Issue by ftomassetti** [smeup/smeup-rpg#2]
These tests have been reported to fail intermittently:
```
com.smeup.rpgparser.execution.RunnerTest > executeExampleWithCall FAILED
    java.lang.AssertionError at RunnerTest.kt:111

com.smeup.rpgparser.execution.RunnerTest > executeExample FAILED
    java.lang.AssertionError at RunnerTest.kt:61

com.smeup.rpgparser.RunnerTest > commandLineProgramsDoesNotRetainStatusOnSetOnLR FAILED
    java.lang.AssertionError at RunnerTest.kt:55

com.smeup.rpgparser.RunnerTest > programsReturnValues FAILED
    java.lang.AssertionError at RunnerTest.kt:24

com.smeup.rpgparser.RunnerTest > commandLineProgramsCanReadSourcesFromString FAILED
    java.lang.AssertionError at RunnerTest.kt:78

com.smeup.rpgparser.RunnerTest > commandLineProgramsCanReadSourcesFromUTF8String FAILED
    java.lang.AssertionError at RunnerTest.kt:95

com.smeup.rpgparser.RunnerTest > commandLineProgramsRetainsStatusOnSetOnRT FAILED
    java.lang.AssertionError at RunnerTest.kt:36

com.smeup.rpgparser.RunnerTest > commandLineProgramCanBeInstrumentedWithAssignmentsLogHandler FAILED
    java.lang.AssertionError at RunnerTest.kt:117
```

**Comment by ftomassetti** [comment]
The tests seem not to be flaky anymore to me, however they seem to regularly fail on the build server while succeeding on my machine. I am exploring this but it is proving to be a quite hard problem to solve.

**Comment by ftomassetti** [comment]
This has been fixed by @f-lombardo in #4
